README.md

```diff
@@ -41,12 +41,7 @@ Then, run the Rails web server:
 
 ### To deploy to App Engine Managed VMs
 
-    $ gcloud preview app deploy app.yaml --set-default
-
-This application uses a custom `Dockerfile` based on the [`google/ruby`][] image.
-
-The `Dockerfile` also runs `bundle install --without development:test` so that only
-production gems will be installed in production.
+    $ gcloud preview app deploy app.yaml --promote
 
 [google/ruby]: https://registry.hub.docker.com/u/google/ruby-runtime/
```

Comment:
(for other branches too) This actually isn't accurate any more :)  We don't use the docker hub image, we push them to gcr.io.  Could we just include a link to the ruby-docker github repo?

github.com/GoogleCloudPlatform/ruby-docker